pkg/dockerclient/shell.go

```diff
@@ -161,17 +161,23 @@ func (c *ShellClient) CopyObjectsToContainer(containerName string, objectsToCopy
 
 		// create target directory if it doesn't exist
 		fileDir := path.Dir(objectContainerPath)
-		if _, err := c.runCommand(nil, "docker exec %s mkdir -p %s", containerName, fileDir); err != nil {
+
+		// escape all paths (security matter)
+		escapedObjectLocalPath := strconv.Quote(objectLocalPath)
+		escapedObjectContainerPath := strconv.Quote(objectContainerPath)
+		escapedFileDir := strconv.Quote(fileDir)
```

Comment:
We have `common.Quote` that is for shell escaping

Reply:
@TomerShor ah, I just noticed that we Compile regexp in `common.Quote`, will also fix this